tenets/codelingo/rfjakob/gocryptfs/lingo_bundle.yaml

```diff
@@ -0,0 +1,6 @@
+tenets:
```

Comment:
Please add description and version. See https://github.com/codelingo/codelingo/blob/master/tenets/codelingo/code-review-comments/lingo_bundle.yaml